playwright-e2e/settings/user-credentials.ts

```diff
@@ -1,8 +1,8 @@
 import dotenv from "dotenv";
 dotenv.config();
 
-const e2ePw = process.env.E2E_TEST_PASSWORD || '';
-const defaultPwd = process.env.SYSTEM_UPDATE_USER_PASSWORD || '';
+const e2ePw = process.env.E2E_TEST_PASSWORD || 'Password1234';
```

Comment:
Hi, do we need the passwords hardcoded?

Reply:
May be not, if it fails to read environment variable in IDE, then add it into your local copy of project only and not commit? Also, fortify scan would catch this as a critical issue I suppose.